invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/meta.html

```diff
@@ -28,6 +28,7 @@
       <meta name="citation_pdf_url" content="{{ file_url }}"/>
     {%- endif %}
     <link rel="alternate" type="{{ file.mimetype }}" href="{{ file_url }}">
```

Comment:
It seems that we do not use the `alternate` type in the right way. According to [MDN's documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel#alternate), it's meant to "indicates that the referenced document is an alternative format (such as a PDF)", but in our case files are other documents than the landing page.

We could remove this line, but on the other hand, people might be relying on it already.